packages/cli/src/commands/server/webSocketProxy.ts

```diff
@@ -54,20 +57,24 @@ function attachToServer(server, path) {
         return;
       }
       debuggerSocket = connection;
-      debuggerSocket.onerror = debuggerSocketCloseHandler;
-      debuggerSocket.onclose = debuggerSocketCloseHandler;
-      debuggerSocket.onmessage = ({data}) => send(clientSocket, data);
+      if (debuggerSocket) {
+        debuggerSocket.onerror = debuggerSocketCloseHandler;
+        debuggerSocket.onclose = debuggerSocketCloseHandler;
+        debuggerSocket.onmessage = ({data}) => send(clientSocket, data);
+      }
     } else if (url.indexOf('role=client') > -1) {
       if (clientSocket) {
-        clientSocket.onerror = null;
-        clientSocket.onclose = null;
-        clientSocket.onmessage = null;
+        clientSocket.onerror = () => {};
+        clientSocket.onclose = () => {};
+        clientSocket.onmessage = () => {};
```

Comment:
I don't think these reassignments are necessary, because they're overwritten right after. It only makes sense to `null` them when `clientSocket.close` throws for some reason and we try to recover – the handlers could leak memory because `clientSocket` would hold them. I'm not sure if that happens, didn't have time to go through this deeply.

For now, let's revert to what was here before and ts-ignore it with a todo comment to rewrite, once we migrate to newer version of ws library.